src/CSSList/CSSList.php

```diff
@@ -296,6 +296,27 @@ public function splice($iOffset, $iLength = null, $mReplacement = null)
         array_splice($this->aContents, $iOffset, $iLength, $mReplacement);
     }
 
+    /**
+     * Insert an item before or after its sibling.
+     *
+     * @param RuleSet|CSSList|Import|Charset $oItem The item.
+     * @param RuleSet|CSSList|Import|Charset $oSibling The sibling.
+     * @param string $sPosition The position.
```

Comment:
For the cases where we already have a (working) native type declaration and where the PHPDoc comment does not add anything substantial over the native annotation, please drop the now-redundant PHPDoc annotation.